src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -3522,10 +3523,10 @@ namespace HVACUnitaryBypassVAV {
                     ZoneLoad = 0.0;
                 }
             } else {
-                ZoneLoad = ZoneSysEnergyDemand(CBVAV(CBVAVNum).ControlledZoneNum(ZoneNum)).RemainingOutputRequired;
+                ZoneLoad = ZoneSysEnergyDemand(actualZoneNum).RemainingOutputRequired;
             }
 
-            if (!CurDeadBandOrSetback(ZoneNum)) {
+            if (!CurDeadBandOrSetback(actualZoneNum)) {
```

Comment:
Fundamental change that corrects zone indexing in the defect file

Reply:
Looks like a great, simple fix